Tools/cases_generator/opcode_metadata_generator.py

```diff
@@ -305,7 +305,8 @@ def generate_pseudo_targets(analysis: Analysis, out: CWriter) -> None:
     table_size = len(analysis.pseudos)
     max_targets = max(len(pseudo.targets) for pseudo in analysis.pseudos.values())
     out.emit("struct pseudo_targets {\n")
-    out.emit(f"uint8_t targets[{max_targets + 1}];\n")
+    out.emit(f"uint8_t as_sequence;\n")
+    out.emit(f"uint8_t targets[{1 + max_targets + 1}];\n")
```

Comment:
Maybe add a comment here: why `1 + max_targets + 1` rather than `max_targets + 2`, and why that size?

Reply:
Oops, we don't need to add the 1+ now. It's a relic from a previous iteration. Now we have the as_sequence field.